services/secure-gateway/config/gateway.json

```diff
@@ -6,21 +6,23 @@
     },
     "security": {
         "enforce": true,
-        "policy_repository": {
-            "upstream_url": "ssh://git@git/tweek/repo",
-            "casbin_policy": "./model.csv",
-            "casbin_model": "./policy.conf"
+        "policyStorage": {
+            "casbinModel": "/model.conf",
+            "casbinPolicy": "policy.csv"
```

Comment:
remove casbinPolicy.csv (unless it's for appending default policy, and that maybe done with env variables)